polaris.shopify.com/content/content/alternative-text.md

```diff
@@ -91,7 +91,7 @@ Always write alt text in plain text. The average rate of listening to a screen r
 - Avoid needless words. If a phrase is still understandable without a specific word, remove it.
 - Remove articles like "a, an, one of," etc. whenever possible. Alt text has different grammatical rules. "Filler words" that assist understanding in speech can get in the way in alt text.
 - Avoid using "image of" or "photograph" unless the type of image is relevant to the context. Screenreaders already announce images with use of the `<img>` attribute.
-- Avoid punctuation like `!!` and emoji like 🥰. Screen readers will announce these as "exclamation point, exclamation point" and "smiling face with three hearts."
+- Avoid punctuation like `!!` and emoji like 🥰. Screen readers will announce these as "exclamation point, exclamation point" and "smiling face with three hearts." Do use other punctuation marks as you regularly would, such as commas or periods at the ends of the sentences. The screen-reader will react with a brief pause.
```

Comment:
Suggestion:
>such as commas, or a period at the end of a sentence

The plural of periods/ends/sentences gets a little confusing, I think. 
And although it's not required, I think a comma after "such as commas" would help clarify that commas don't go at the end of a sentence—just periods.